### PR TITLE
Added Workflow Trigger

### PR DESCRIPTION
**Pull request by Evan8456**
- Added an actions file that would send a http trigger to sensein/b2ai-redcap2rs for redcap to reproschema automation.
- Requires a PAT to be added in the secrets in order for the trigger to be accepted.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Trigger workflow to build ReproSchema from RedCap
+        run: |
+          curl --request POST \
+          --header "Authorization: token ${{ secrets.B2AI_REDCAP2RS_PAT }}" \
```
Comment (alexsigaras) [inline .github/workflows/deploy.yml:16]:
```suggestion
          --header "Authorization: token ${{ secrets.B2AIPREP_PAT }}" \
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Trigger ReproSchema to RedCap workflow
```
Comment (alexsigaras) [inline .github/workflows/deploy.yml:1]:
```suggestion
name: Trigger ReproSchema to REDCap workflow
```

**Review by alexsigaras — CHANGES_REQUESTED** [review]
Hi @Evan8456,

Could you please make these minor changes? The most important one is to reflect the correct name of the PAT.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (Evan8456) [reply .github/workflows/deploy.yml:16]: The PAT is for the reposoitory redcap2rs(https://github.com/sensein/b2ai-redcap2rs) is is separate from the b2aiprep repository.

**Review by alexsigaras — APPROVED** [review]
🚀

**Pull request merged**